src/cljc/witan/schema.cljc

```diff
@@ -46,6 +46,11 @@
    (s/required-key :type)    ModelPropertyType
    (s/optional-key :context) s/Any}) ;; varies depending on the type
 
+(def ModelPropertyValue
+  "A model property and value binding"
+  {(s/required-key :property-id) IdType
+   (s/required-key :value) s/Any}) ;; varies depending on the ModelProperty type
```

Comment:
looks legit.
If we want to make the best possible use of schema validation (not using s/Any), we could have a ModelPropertyValue type for every type of ModelProperty.  In that case we could go quite far - tbd